GeneratorInterface/Pythia8Interface/test/pythia8ex12_cfg.py

```diff
@@ -30,7 +26,11 @@
               'SpaceShower:pTmaxMatch = 2',
               'TimeShower:pTmaxMatch = 2',
         ),
-        pythia8_example12 = cms.vstring('POWHEGres:calcScales = on'),
+        pythia8_example12 = cms.vstring(
+              'POWHEGres:calcScales = on',
+              'Beams:frameType = 4',
+              'Beams:LHEF = powheg-b_bbar_4l.lhe',
```

Comment:
It is on purpose, so that the LHE file is read by Pythia, not by the CMSSW LHEReader